tlovett1/safe-redirect-manager
# Check non-active multisite directory against the main site redirects

### Description of the Change

This PR contains fixes for this issue: https://github.com/10up/safe-redirect-manager/issues/212. Whenever the multisite subsite is not active, check the requested path against the saved redirect rules from the main website.

### Verification Process

- Create a multisite installation with several sub-sites.
- Archive, deactivate or mark a site as spam
- Go to the main website and add a redirect rule for the subsite you've modified the status from step 2.
- Verify that the redirect is working.

### Checklist:






### Applicable Issues



### Changelog Entry
